ts/features/walletV3/onboarding/types/index.d.ts

```diff
@@ -0,0 +1,40 @@
+export type WalletOnboardingStartResponse = { id: number; redirectUrl: string };
```

Comment:
Naming a file with `.d.ts` extension means that you are declaring global types. Please avoid it, it could lead to problems.

Reply:
Addressed in [2d50633](https://github.com/pagopa/io-app/pull/4810/commits/2d5063304f55de1653688acedb5a5b1d3beeabc3)